src/analytics/ValoraAnalytics.ts

```diff
@@ -109,6 +110,9 @@ class ValoraAnalytics {
         writeKey: SEGMENT_API_KEY,
         storePersistor: AsyncStoragePersistor,
       })
+      await LegacySegment.setup('n/a', {})
+      const anonymousId = await LegacySegment.getAnonymousId()
```

Comment:
I think it might be overkill to import LegacySegment just to do this. Can we import NativeModules from react-native and call `NativeModules.RNAnalytics.getAnonymousId()` here instead?

Reply:
`RNanalytics` is actually a custom native module made by segment inside of the legacy `@segment/react-native-analytics` package. So we would have to re-implement large parts of that to replace it

Reply:
dang